presto-docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -89,6 +89,32 @@ Run ``SELECT`` to access the ``cks`` table in the ``tpch`` database::
     If you used a different name for your catalog properties file, use
     that catalog name instead of ``clickhouse`` in the above examples.
 
+PrestoDB to ClickHouse Type Mapping
+-----------------------------------
+
+========================================== ========================= =====================================================================================================
+**PrestoDB Type**                          **ClickHouse Type**       **Notes**
+========================================== ========================= =====================================================================================================
+BOOLEAN                                    UInt8                     ClickHouse uses UInt8 as boolean, restricted values to 0 and 1.
+TINYINT                                    Int8
+SMALLINT                                   Int16
+INTEGER                                    Int32
+BIGINT                                     Int64
+REAL                                       Float32
+DOUBLE                                     Float64
+DECIMAL                                    Decimal(precision, scale) The precision and scale are dynamic based on the PrestoDB type.
+CHAR / VARCHAR                             String                    The String type replaces VARCHAR, BLOB, CLOB, etc., from other DBMSs.
+VARBINARY                                  String
+DATE                                       Date
+TIMESTAMP                                  DateTime64(3)             Timestamp with 3 digits of millisecond precision. This is tested against Clickhouse-Server 23.12.2.59
```

Comment:
I would recommend leaving out last sentence with the clickhouse server version. It is unlikely this doc will be updated if we upgrade the test server in the future.